book/zappifest/plugins-manifest-format.md

```diff
@@ -355,6 +355,23 @@ Field Key         | Description
 ###### General Fields:
 - **ios_font_selector**: Will populate fonts dropdown with iOS system fonts and Zapp app custom uploaded fonts
 - **android_font_selector**: Will populate fonts dropdown with Android system fonts and Zapp app custom uploaded fonts
+- **samsung_tv_font_selector**: Will populate fonts dropdown with Samsung TV system fonts and Zapp app custom uploaded fonts
```

Comment:
In my opinion, we need to write here from which version it's supported. this is relevant for all selectors.

Reply:
its depend on plugin... not the selector, selector sending only a string value